packages/server/graphql/public/mutations/upsertTeamPromptResponse.ts

```diff
@@ -0,0 +1,81 @@
+import {JSONContent} from '@tiptap/core'
+import {SubscriptionChannel} from 'parabol-client/types/constEnums'
+import {upsertTeamPromptResponse as upsertTeamPromptResponseQuery} from '../../../postgres/queries/upsertTeamPromptResponses'
+import {getUserId, isTeamMember} from '../../../utils/authorization'
+import publish from '../../../utils/publish'
+import standardError from '../../../utils/standardError'
+import extractTextFromTipTapJSONContent from '../../mutations/helpers/tiptap/extractTextFromTipTapJSONContent'
+import {MutationResolvers} from '../resolverTypes'
+
+const upsertTeamPromptResponse: MutationResolvers['upsertTeamPromptResponse'] = async (
+  _source,
+  {teamPromptResponse},
+  {authToken, dataLoader, socketId: mutatorId}
+) => {
+  const viewerId = getUserId(authToken)
+  const operationId = dataLoader.share()
+  const subOptions = {mutatorId, operationId}
+  const {teamPromptResponseId, meetingId, content} = teamPromptResponse
+
+  // AUTH
+  const promptResponse = teamPromptResponseId
+    ? await dataLoader.get('teamPromptResponses').loadNonNull(teamPromptResponseId)
```

Comment:
-2 if I see correctly it'll throw an error if there's no data, so we'd need to use normal `load` method here